### DISCUSSION TITLE
Allow quoted file names in FileTree

### DISCUSSION THREAD

**Discussion by iskandersierra** [withastro/starlight#2608] [Feature Requests]
### What version of `starlight` are you using?

0.28.3

### What is your idea?

Allow using file name with spaces by using quotes, like `"my file"` o `'my folder'`.

### Why is this feature necessary?

Linux and Windows allow using spaces in files and folder names, but I have not found any way to represent that in the `FileTree`.

Currently, if I use quotes like `- "Solution Items" misc files` the tree represents `"Solution` as the file name, and `Items"` as the comments

### Do you have examples of this feature in other projects?

```jsx
<FileTree>
- "Solution Items"  misc files
- "my tests"  test modules
</FileTree>
```

Would show:
- **Solution Items**  _misc files_
- **my tests**  _test modules_

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Hi @iskandersierra! I’ve been waiting for the day someone would notice and request this 😁 

I think supporting this by wrapping with `"` like you propose makes sense and it seems unlikely to cause issues as file names that start and end with `"` seem even more rare than names containing spaces.

Would be happy to take a look at a PR if you can put one together!

**Reply by iskandersierra** [reply]
I'll look into it ASAP. I'm still not familiar with the source code.
